atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -44,6 +44,28 @@
 
     @Value.Default
     public int poolSize() {
+        return 30;
```

Comment:
Personally I prefer explicitly setting the min & max to control resource utilization.